deltachat-ios/ViewModel/ChatListViewModel.swift

```diff
@@ -228,25 +228,25 @@ class ChatListViewModel: NSObject {
         INInteraction.delete(with: ["\(dcContext.id).\(chatId)"])
     }
 
-    func archiveChatToggle(chatId: Int) {
+    func archiveChatToggle(chatId: Int, notifyListener: Bool = false) {
```

Comment:
unless it is on purpose, the `notifyListener` parameter can be dropped, it is always true. same for `pinChatToggle`.